src/pages/index.js

```diff
@@ -1,57 +1,107 @@
 import FormValidator from "../components/FormValidator.js";
-import { initialCards, settings } from "../utils/constants.js";
+import { settings } from "../utils/constants.js";
 import Card from "../components/Card.js";
 import "./index.css";
 import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithForm from "../components/PopupWithForm.js";
-("../components/PopupWithForm.js");
+import Api from "../components/Api.js";
+import PopupConfirmation from "../components/PopupConfirmation.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "967eb4d1-2e88-4a75-bdf4-9be2dee67768",
+    "Content-Type": "application/json",
+  },
+});
 
-/* Elements */
+api.getProfileInfo().then(({ name, about, avatar }) => {
+  userInfo.setUserInfo({ name, about });
+  userInfo.setAvatar({ avatar });
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**